pkg/modules/registry/registry.go

```diff
@@ -0,0 +1,64 @@
+package registry
```

Comment:
@wbrowne pulled the registry into https://github.com/grafana/grafana/pull/64062, so any comments on this should be redirected to that PR since it's targeting `main`